components/cli/pkg/commands/project/init_test.go

```diff
@@ -54,7 +55,7 @@ func TestRunInit(t *testing.T) {
 	}
 	for _, tst := range tests {
 		t.Run(tst.name, func(t *testing.T) {
-			err := RunInit(mockCli, tst.project, "")
+			err := RunInit(mockCli, tst.project, tst.isProject)
```

Comment:
Better if other scenarios in init command could be covered. i.e "initTest" function etc.

Reply:
Will be added by another PR. Created #990 to track the progress.